dirty_cat/similarity_encoder.py

```diff
@@ -157,7 +158,9 @@ class SimilarityEncoder(OneHotEncoder):
         HashingVectorizer with a number of features equal to `hashing_dim`.
     n_prototypes: number of prototype we want to use.
         Useful when `most_frequent` or `k-means` is used.
-        Must be a positive non null integer.
+        Must be a positiv non null integer.
```

Comment:
Typo

Reply:
@GaelVaroquaux I think I got all the merge errors. If it's good for you we can merge it, then merge this into my other PR and rerun the benchmark and tests.